systemtest/otlp_test.go

```diff
@@ -206,7 +206,7 @@ func TestOTLPGRPCMetrics(t *testing.T) {
 
 	// opentelemetry-go does not support sending Summary metrics,
 	// so we send them using the lower level OTLP/gRPC client.
-	conn, err := grpc.Dial(serverAddr(srv), grpc.WithInsecure(), grpc.WithBlock())
+	conn, err := grpc.Dial(serverAddr(srv), grpc.WithInsecure(), grpc.WithBlock(), grpc.WithDefaultCallOptions(grpc.UseCompressor("gzip")))
```

Comment:
Could there be a future regression where it's requests without gzip that don't work? Should we also test without gzip enabled?